src/pages/content-edit/lib/delegator-validity.js

```diff
@@ -0,0 +1,154 @@
+import m from "mithril";
+import debounce from "lodash.debounce";
+
+import css from "../invalid-msg.css";
+
+function reqPrefix(name) {
+    return "Required: " + name;
+}
+
+export default function Validity(content) {
+    this.handlersAttached = false;
+    this.content = content;
+}
+
+Validity.prototype = {
+
+    // Private functions.
```

Comment:
I tend not to use `prototype`s unless really necessary, which explains why it doesn't come up much.